src/utils/config.ts

```diff
@@ -70,7 +70,19 @@ export const MAXIMUM_DRAIN_DELAY = Math.round(
 	(1 * MINUTES) / intVar('MINIMUM_DRAIN_RATE_PER_MINUTE', 500),
 );
 
-export const VPN_INSTANCE_COUNT = getInstanceCount('VPN_INSTANCE_COUNT');
+export let VPN_INSTANCE_COUNT = getInstanceCount('VPN_INSTANCE_COUNT');
```

Comment:
my understanding is that this export will not update if the value changes below, but someone else could confirm (or we could write a small test)

Reply:
It should be fine, especially with ESM as open-balena-vpn uses (but even with typescript+commonjs it should be ok due to how it transpiles, it's plain commonjs that could be problematic in some cases)